rslib/src/card/mod.rs

```diff
@@ -490,6 +496,9 @@ impl From<MemoryState> for FsrsMemoryState {
 
 #[cfg(test)]
 mod test {
+    use crate::prelude::AnkiError;
+    use crate::prelude::Collection;
+    use crate::prelude::DeckId;
```

Comment:
It's more idiomatic to import `crate::prelude::*` and/or `super::*`

Reply:
Fixed, thank you!